dash/backend/src/modules/command-line/commands/falco-email.command.ts

```diff
@@ -0,0 +1,40 @@
+import {Injectable} from "@nestjs/common";
+import {MineLoggerService} from "../../shared/services/mine-logger.service";
+import {KubernetesHistoryService} from "../services/kubernetes-history.service";
+import {ClusterDao} from "../../cluster/dao/cluster.dao";
+import {PodComplianceService} from "../../pod/services/pod-compliance.service";
+import {ImageRescanningService} from "../services/image-rescanning.service";
+import {NamespaceComplianceService} from "../../namespace/services/namespace_compliance.service";
+import {Command} from "nestjs-command";
+import {yesterdaysDateAsStr} from "../../../util/date_util";
+import {FalcoService} from "../../falco/service/falco.service";
+
+/**
+* This command is to schedule to send falco SUMMARY emails
+*/
+@Injectable()
+export class FalcoEmailCommand {
+
+    constructor(private readonly loggerService: MineLoggerService,
+                private readonly falcoService: FalcoService,
+                ) {}
+
+    @Command({ command: 'schedule:falco-email', describe: 'Schedule Falco email' })
+    async get() {
+        /*const clusters = await this.clusterService.getClusters();
```

Comment:
Lets add a //@TODO comment explaining that this feature hasn't been built yet, that way if someone is reading the code they aren't confused about why it is here and commented out.